app/errors.js

```diff
@@ -3,8 +3,16 @@ const internalError = (message, internalCode) => ({
   internalCode
 });
 
+const externalError = (message, externalCode) => ({
+  message,
+  externalCode
+});
+
 exports.DATABASE_ERROR = 'database_error';
 exports.databaseError = message => internalError(message, exports.DATABASE_ERROR);
 
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
+
+exports.API_ERROR = 'api_error';
+exports.apiError = message => externalError(message, exports.API_ERROR);
```

Comment:
You are forgetting to add this to the errors middleware to assign it a status code. For choosing the status code itself, have a look at https://en.wikipedia.org/wiki/List_of_HTTP_status_codes